### PR TITLE
DEVDOCS-2491: Added ShopperLanguageSelectionMethod to settings/store/locale

### PR DESCRIPTION
**Pull request by ektravel**
Added ShopperLanguageSelectionMethod to settings/store/locale

### CODE REVIEW

File: reference/settings.v3.yml [hunk 1]
```diff
@@ -924,6 +924,21 @@ components:
         store_country:
           type: string
           default: United States
+        shopper_language_selection_method:
+          type: string
+          enum:
+            - browser
+            - default_shopper_language
+          example: browser
+          default: browser
```
Comment (kristinapototska) [inline reference/settings.v3.yml:933]: default value would be a `default_shopper_language`

Comment (kristinapototska) [reply reference/settings.v3.yml:933]: I guess example would also be better to provide for `default_shopper_language` selected as a default value for `shopper_language_selection_method`

File: reference/settings.v3.yml [hunk 2]
```diff
@@ -924,6 +924,21 @@ components:
...
+          description: |-
+            Determines whether to display the storefront content in the shopper's browser language or the shopper's selected default language.
```
Comment (kristinapototska) [inline reference/settings.v3.yml:935]: > Determines whether to display the storefront content in the shopper's browser language(s) or the selected single shopper's default language.

I think it's good to callout here that with shopper browser language multiple languages are supported, but with default shopper language – it's gonna be single selected language

**Review by traci707 — COMMENTED** [review]
Just resolve Kristina's changes and it looks good to me.

File: reference/settings.v3.yml [hunk 3]
```diff
@@ -924,6 +924,21 @@ components:
...
+            - browser
+            - default_shopper_language
+          example: default_shopper_language
+          default: default_shopper_language
+          description: |-
+            Determines whether to display the storefront content in the shopper's browser language or the shopper's selected default language.
+
+            Available values: 
+
+            `browser` - language is set automatically based on the shopper browser language. Multiple languages are supported.
```
Comment (traci707) [inline reference/settings.v3.yml:939]:
```suggestion
            `browser` - language updates automatically based on the shopper browser language. Multiple languages are supported.
```

**Review by traci707 — COMMENTED** [review]
I found one small change.

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**